src/state/slices/txHistorySlice/selectors.ts

```diff
@@ -173,3 +179,42 @@ export const selectRebasesByFilter = createSelector(
   selectRebaseIdsByFilter,
   (rebasesById, rebaseIds) => rebaseIds.map(rebaseId => rebasesById[rebaseId]),
 )
+
+/**
+ * to be able to add an account for a chain, we want to ensure there is some tx history
+ * on the current highest accountNumber accountIds
+ *
+ * note - there can be multiple accountIds sharing the same accountNumber - e.g. BTC legacy/segwit/segwit native
+ * are all separate accounts that share the same account number
+ */
+export const selectMaybeNextAccountNumberByChainId = createSelector(
+  selectTxIdsByAccountId,
+  selectPortfolioAccountMetadata,
+  selectChainIdParamFromFilter,
+  (txIdsByAccountId, accountMetadata, chainId): [boolean, number] => {
+    // filter accounts by chain id
+    const accountMetadataEntriesByChainId: [AccountId, AccountMetadata][] = Object.entries(
+      accountMetadata,
+    ).filter(([accountId, _metadata]) => fromAccountId(accountId).chainId === chainId)
+
+    // grab the highest account number
+    const currentHighestAccountNumber: number = Math.max(
+      ...accountMetadataEntriesByChainId.map(([, { bip44Params }]) => bip44Params.accountNumber),
+    )
+
+    // filter for highest account number, and map back to accountIds
+    const highestAccountNumberAccountsIds: AccountId[] = accountMetadataEntriesByChainId
+      .filter(
+        ([_accountId, { bip44Params }]) =>
+          bip44Params.accountNumber === currentHighestAccountNumber,
+      )
+      .map(([accountId]) => accountId)
+
+    // at least one of the account ids with the highest account number must have some tx history
+    const isAbleToAddNextAccount = highestAccountNumberAccountsIds.some(accountId =>
+      Boolean((txIdsByAccountId[accountId] ?? []).length),
+    )
+    const nextAccountNumber = currentHighestAccountNumber + 1
+    return [isAbleToAddNextAccount, nextAccountNumber]
```

Comment:
change this to `[true, nextAccountNumber]` to test adding more than account 1 without transacting on it